troposphere/static/js/components/modals/link/ExternalLinkDeleteModal.react.js

```diff
@@ -8,7 +8,7 @@ export default React.createClass({
     mixins: [BootstrapModalMixin],
 
     propTypes: {
-      link: React.PropTypes.instanceOf(Backbone.Model).isRequired
+          external_link: React.PropTypes.instanceOf(Backbone.Model).isRequired
```

Comment:
This is using Python variable naming. I'm not sure if we want to deal with renaming the JSX attribute usage to camelcase, externalLink. It's kind of trivial. Not sure how you feel.